styles/styles.css

```diff
@@ -1166,6 +1164,11 @@ main .section.with-background-image.align-top > .section-container, main .sectio
     margin-bottom: 1rem;
   }
 
+  /* section - static */
+  main .section.static > .section-container {
```

Comment:
are we using static only at one place ?

Reply:
yes, that's what I found atleast through finder.

Reply:
Also similar observation with Sharepoint search. There's also a `hero-banner(static)` variant but likely that's not related to this change as this one is limiter to section level style.